discovery-infra/tests/config/global_configs.py

```diff
@@ -14,6 +15,10 @@
 global_variables = GlobalVariables()
 
 
+log.debug(f"global_variables={global_variables}")
+log.debug(f"environ={os.environ}")
```

Comment:
seems a bit messy to have this kind of log on module level
can we add it anywhere else?